### PR TITLE
Fix team page in Edge

### PR DESCRIPTION
**Pull request by sophieH29**
related to this issue https://github.com/gdg-x/hoverboard/issues/106

### CODE REVIEW

File: src/pages/team-page.html [hunk 1]
```diff
@@ -214,9 +213,9 @@ <h3 class="team-name" cascaded>[[team.title]]</h3>
           <div class="team-container">
             <template is="dom-repeat" items="[[team.members]]" as="member" on-dom-change="_updateCascadedNodes">
               <div class="member">
-                <paper-material class="card" cascaded>
+                <paper-material class="card"  cascaded>
                   <div class="header">
-                    <div class="photo" style="background-image: url('[[member.photoUrl]]')"></div>
+                    <img class="photo" src="[[member.photoUrl]]"/>
```
Comment (ozasadnyy) [inline src/pages/team-page.html]: Use iron-image instead

**Comment by ozasadnyy** [comment]
Just replace `img` with `iron-image` and we will be done. Great job!

File: src/pages/team-page.html (hunk 1 above)
Comment (sophieH29) [inline src/pages/team-page.html]: why? what the problem with regular img element?

Comment (ozasadnyy) [inline src/pages/team-page.html]: Preload and sizing property

Comment (MrCsabaToth) [inline src/pages/team-page.html]: Like `<iron-image src="[[member.photoUrl]]" class="photo placeholder-color" preload="" fade="" sizing="cover"></iron-image>`

Comment (ozasadnyy) [inline src/pages/team-page.html]: if it's a boolean value you can write `.. preload fade sizing="cover" ..`

**Comment by ozasadnyy** [comment]
After the conversation with @sophieH29 we decided to keep `img` for now because there is a bug in `iron-image` with [unmerged PR](https://github.com/PolymerElements/app-storage/pull/45/commits/7ae41566bd186199ed9f05340fd08fc97f8b3d16).

File: src/pages/team-page.html (hunk 1 above)
Comment (MrCsabaToth) [inline src/pages/team-page.html]: Oh, I wanted to say I got that from the https://github.com/gdg-x/hoverboard/blob/develop/src/pages/speakers-page.html#L183 but you are right!

**Comment by MrCsabaToth** [comment]
Thanks for the info!

**Pull request merged**